[2024-10-28T22:08:31.844+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-28T22:08:31.905+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: indices_provincias_v01.inflacion_parana scheduled__2024-10-27T00:00:00+00:00 [queued]>
[2024-10-28T22:08:31.935+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: indices_provincias_v01.inflacion_parana scheduled__2024-10-27T00:00:00+00:00 [queued]>
[2024-10-28T22:08:31.936+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-10-28T22:08:31.975+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): inflacion_parana> on 2024-10-27 00:00:00+00:00
[2024-10-28T22:08:32.006+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'indices_provincias_v01', 'inflacion_parana', 'scheduled__2024-10-27T00:00:00+00:00', '--job-id', '236', '--raw', '--subdir', 'DAGS_FOLDER/second_dag.py', '--cfg-path', '/tmp/tmp164kn7kk']
[2024-10-28T22:08:32.018+0000] {standard_task_runner.py:105} INFO - Job 236: Subtask inflacion_parana
[2024-10-28T22:08:32.024+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=3618) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-28T22:08:32.025+0000] {standard_task_runner.py:72} INFO - Started process 3621 to run task
[2024-10-28T22:08:32.196+0000] {task_command.py:467} INFO - Running <TaskInstance: indices_provincias_v01.inflacion_parana scheduled__2024-10-27T00:00:00+00:00 [running]> on host 276004bbea2b
[2024-10-28T22:08:32.745+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='rolo' AIRFLOW_CTX_DAG_ID='indices_provincias_v01' AIRFLOW_CTX_TASK_ID='inflacion_parana' AIRFLOW_CTX_EXECUTION_DATE='2024-10-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-27T00:00:00+00:00'
[2024-10-28T22:08:32.749+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-28T22:09:03.449+0000] {logging_mixin.py:190} INFO - Error: [CANNOT_INFER_TYPE_FOR_FIELD] Unable to infer the type of the field `fecha_inflacion`.
[2024-10-28T22:09:04.048+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-28T22:09:04.063+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-28T22:09:04.064+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=indices_provincias_v01, task_id=inflacion_parana, run_id=scheduled__2024-10-27T00:00:00+00:00, execution_date=20241027T000000, start_date=20241028T220831, end_date=20241028T220904
[2024-10-28T22:09:04.132+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-28T22:09:04.165+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-28T22:09:04.167+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
